src/components/Layout/TableOfContents.jsx

```diff
@@ -60,58 +61,50 @@ const EntryTitle = styled.h6`
   line-height: 1.5;
   border-bottom: 1px solid transparent;
   text-decoration: none;
-`
+  `
 
 const ChapterListItem = styled.li`
   margin: 0;
-`
+  `
 
 const EntryListItem = styled.li`
   margin: 0;
-  a:hover {
+  a:hover
+  {
     border-bottom: 1px solid black;
   }
-`
+  `
 
 const ChapterTitle = styled.h5`
-  font-weight: ${({ level }) => {
-    switch (level % 3) {
-      case 1:
-        return '600'
-      case 2:
-        return '400'
-      default:
-        return '200'
-    }
-  }};
-  font-size: ${({ level }) => {
-    switch (level % 3) {
-      case 1:
-        return '2.4rem'
-      case 2:
-        return '2rem'
-      default:
-        return '2.8rem'
-    }
-  }};
-  color: ${({ level, theme }) => {
-    switch (level % 3) {
-      case 1:
-        return 'black'
-      case 2:
-        return 'blue'
-      default:
-        return theme.brand
-    }
-  }};
-  margin-bottom: ${({ level }) => {
-    switch (level % 3) {
-      case 1:
-        return '7px'
-      case 2:
-        return '5px'
-      default:
-        return '10px'
-    }
-  }};
+  font - weight
+: ${({level}) => {
+  switch (level % 3) {
+    case 1:
+      return '600'
+    case 2:
+      return '400'
+    default:
+      return '200'
+  }
+}};
+  font-size: ${({level}) => {
+  switch (level % 3) {
+    case 1:
+      return '2.2rem'
```

Comment:
Is this change about making it "nicer"? Maybe it is better to make the difference between the headlines more visible than nice. People will change these values anyhow if they really care about "nice".